packages/express/README.md

```diff
@@ -0,0 +1,467 @@
+# Transifex Native integration for Express JS
+
+## Quick start
+
+Install the necessary express packages:
+
+```shell
+npm install --save express cookie-parser body-parser ...
+```
+
+And the Transifex Native integration:
+
+```shell
+npm install --save @transifex/express
```

Comment:
Since you declare `@transifex/native` as a peerDependency, this will work only on NPM7. NPM prior to 7, does NOT automatically install peer dependencies, so this will not work. Two options:
- Add `@transifex/native` as dependency 
- Update the example here to `npm install --save @transifex/native @transifex/express`